docs/content/en/docs/topics/restore-a-backup/amazon-s3/_index.md

```diff
@@ -0,0 +1,398 @@
+{{% /alert %}}
+
+In this example: `demo-superuser`. To verify, you can extract and decode the username secret:
+
+ ```bash
+ kubectl get secret demo-superuser -o jsonpath="{.data.username}" | base64 --decode ; echo
+ ```
+
+ **Output:**
+
+ ```bash
+ demo-superuser
+ ```
+
+The output verified that the username to use on a subsequent command is `demo-superuser`.
+
+Next, extract and decode the password secret. For example:
+
+ ```bash
+ kubectl get secret demo-superuser -o jsonpath="{.data.password}" | base64 --decode ; echo
+ ```
+
+ **Output:**
+
+ ```bash
+ ItSCody8Nou2R0vYWd4FPDGi2RPy-Jvtg-mSeOoUlbU5UNsIlDF8QQ
+ ```
+
+{{% alert title="Tip" color="success" %}}
+The password above is an example. The value will be different for your environment. In the subsequent examples, don’t forget to replace the sample password with the one extracted in your environment.
+{{% /alert %}}
+
+### Use the credentials and add the data to the Cassandra database:
+
+For example:
+
+```bash
+kubectl exec -it demo-dc1-default-sts-0 -c cassandra -- cqlsh -u demo-superuser -p ItSCody8Nou2R0vYWd4FPDGi2RPy-Jvtg-mSeOoUlbU5UNsIlDF8QQ -f /tmp/test_data.cql
+```
+
+### Exec open CQLSH and enter DML and DDL statements
+
+```bash
+kubectl exec -it demo-dc1-default-sts-0 -c cassandra -- cqlsh -u demo-superuser -p ItSCody8Nou2R0vYWd4FPDGi2RPy-Jvtg-mSeOoUlbU5UNsIlDF8QQ
+ 
+Connected to demo at 127.0.0.1:9042.
+[cqlsh 5.0.1 | Cassandra 3.11.10 | CQL spec 3.4.4 | Native protocol v4]
+Use HELP for help.
+demo-superuser@cqlsh> USE medusa_test;
+demo-superuser@cqlsh:medusa_test> SELECT * from medusa_test.users;
+
+ email             | name          | state
+-------------------+---------------+-------
+ alice@example.com |   Alice Smith |    TX
+   bob@example.com |     Bob Jones |    VA
+ david@example.com |    David Yang |    NV
+ carol@example.com | Carol Jackson |    CA
+
+(4 rows)
+```
+
+Exit out of CQLSH:
+
+```bash
+demo-superuser@cqlsh:medusa_test> quit;
+```
+
+{{% alert title="Tip" color="success" %}}
+Keep the sample `medusa_test.users` keyspace.table data in mind -- we will use subsequent backup and restore steps with this data.
+{{% /alert %}}
+
+### Create the backup
+
+Now create a backup by referencing the backup chart:
+
+```bash
+helm install test k8ssandra/backup --set name=test,cassandraDatacenter.name=dc1
+```
+
+**Output:**
+
+```bash
+kubectl get cassandrabackup
+NAME       AGE
+test       17s
+```
+
+Examine the YAML:
+
+```bash
+kubectl get cassandrabackup test -o yaml
+```
+
+The Status section in the YAML shows the backup operation’s start and finish timestamps.
+
+### Amazon S3 buckets
+
+Let's look at the resources in the Amazon S3 dashboard.
+
+S3 maintains the `backup_index` bucket so it only has to store a single copy of an SSTable across backups.  S3 stores pointers in the index to the SSTables. That implementation avoids a large amount of storage.  For example:
```

Comment:
That's slightly inaccurate: it's Medusa that maintains a backup index as top level folder in the bucket, to make operations such as listing backups faster (you don't need to scan the whole bucket but just the index). The SSTables being copied only once is part of the differential backup feature (which is the default mode) that checks the manifest from the previous backup to skip SSTables that were already present there.
The manifest from the newly created backups will then point to both pre-existing sstables and newly uploaded ones.

Reply:
Thanks for the helpful comments - edited the paragraph.